docs/mobility/motion/_index.md

```diff
@@ -537,7 +544,211 @@ myMvmntSensorResourceName := movement_sensor.Named("my_movement_sensor")
+- `timeout` [(Optional\[float\])](https://docs.python.org/library/typing.html#typing.Optional): An option to set how long to wait (in seconds) before calling a time-out and closing the underlying RPC call.
+
+**Returns:**
+
+- [(GetPlanResponse)](https://python.viam.dev/autoapi/viam/services/motion/index.html#viam.services.motion.GetPlanResponse)
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/services/motion/index.html#viam.services.motion.MotionClient.get_plan).
+
+```python {class="line-numbers linkable-line-numbers"}
+motion = MotionClient.from_robot(robot=robot, name="builtin")
+my_base_resource_name = Base.get_resource_name("my_base")
+# Get the plan(s) of the base component which was instructed
+# to move by `MoveOnGlobe()`
+resp = await motion.get_plan(component_name=my_base_resource_name)
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `ctx` [(Context)](https://pkg.go.dev/context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+- `req` [PlanHistoryReq](https://pkg.go.dev/go.viam.com/rdk/services/motion@v0.15.1#PlanHistoryReq): A `PlanHistoryReq` which contains the following values:
+  - `componentName` [(resource.Name)](https://pkg.go.dev/go.viam.com/rdk/resource#Name): The `resource.Name` of the base to stop.
+  - `lastPlanOnly` [(bool)](https://pkg.go.dev/builtin#bool): If `true`, the response will only return the the last plan for the component / execution
+  - `executionID` [(ExecutionID)](https://pkg.go.dev/go.viam.com/rdk/services/motion@v0.15.1#PlanHistoryReq): If non empty, the response will return the plans of the provided execution & component. Useful for retrieving plans from executions before the current execution.
+  - `extra` [(map\[string\]interface{})](https://go.dev/blog/maps): Extra options to pass to the underlying RPC call.
+
+**Returns:**
+
+- [([]PlanWithStatus)](https://pkg.go.dev/go.viam.com/rdk/services/motion@v0.15.1#PlanWithStatus): PlanWithStatus contains a plan, its current status, and all state changes that came prior sorted by ascending timestamp.
+- [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/services/motion#Service).
+
+```go {class="line-numbers linkable-line-numbers"}
+motionService, err := motion.FromRobot(robot, "builtin")
+# Get the plan(s) of the base component's most recent execution i.e. `MoveOnGlobe()` call.
+ctx := context.Background()
+planHistory, err := motionService.PlanHistory(ctx, motion.PlanHistoryReq{
+    ComponentName: s.req.ComponentName,
+})
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### ListPlanStatuses
+
+Returns the plan statuses created by [`MoveOnGlobe`](/mobility/motion/#moveonglobe) calls that are executing OR are part of an execution which changed it state within the a 24HR TTL OR until the robot reinitializes.
```

Comment:
```suggestion
Returns the plan statuses created by [`MoveOnGlobe`](/mobility/motion/#moveonglobe) calls that meet at least one of the following conditions:

 - currently in execution
 - part of an execution which changed its state within the last 24 hours while the machine hasn't reinitialized (for example, due to a restart).
```

Reply:
This would be more accurate:
```
Returns the statuses of plans created by [`MoveOnGlobe`](/mobility/motion/#moveonglobe) calls that met at least one of the following conditions since the motion service initialized:

- the plan's status is in progress
- the plan's status changed state within the last 24 hours
```